Soccar [Game-Server]/src/nl/soccar/gameserver/controller/rmi/GameServerRmiController.java

```diff
@@ -26,24 +29,54 @@
     private IMainServerForGameServer mainServerForGameServer;
     private GameServerForMainServer gameServerForMainServer;
 
+    /**
+     * Initializes the GameServerRmiController class.
+     * 
+     * @param host The given host, not null.
+     * @param port The given port, not null.
+     */
     public GameServerRmiController(String host, int port) {
         this.host = host;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this method to throw at most one checked exception instead of: java.rmi.NotBoundException, java.rmi.RemoteException [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AS1160)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)